primitives/src/migrate.rs

```diff
@@ -21,6 +21,15 @@ use frame_support::storage::unhashed::kill_prefix;
 use frame_support::{ReversibleStorageHasher, StorageHasher, Twox128};
 use sp_std::vec::Vec;
 
+/// A migration error type.
+#[derive(Clone, PartialEq, Eq, Encode, Decode, Debug)]
+pub enum MigrationError<T: Encode + Decode> {
```

Comment:
What happens if you remove the two remaining bounds?

Reply:
You're right, we do not need them.